mne/viz/topomap.py

```diff
@@ -1485,11 +1485,8 @@ def plot_evoked_topomap(evoked, times="auto", ch_type=None,
         Add markers for sensor locations to the plot. Accepts matplotlib plot
         format string (e.g., 'r+' for red plusses). If True (default),
         circles will be used.
-    colorbar : bool | None
+    colorbar : bool
```

Comment:
I didn't think so initially, because previously `None` just got converted to `True` automatically, and it was the default. But I suppose this *might* break user code if anyone had explicitly passed `colorbar=None`.  I'll fix this.